Todo/JS/TodoServerSide/TodoServer_DL/RamDB.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using TodoServer_Commons;
+namespace TodoServer_DL
+{
+    public class RamDB : TodoServer_Commons.IDB
+    {
+        private List<TodoTask> _tasks;
+
+        public RamDB()

```

Comment:
ramdb == cache
using a cache is good because you dont need to get data from the db, but because its all stored in memory it shouldnt be your only data source (you can write to file, use jsonserver or whatever)